### PR TITLE
adjust table margin bottom

### PR DESCRIPTION
**Pull request by kbhawkey**
Remove td-main table custom margin settings.
There is inadequate spacing between text that immediately follows a table element.

### CODE REVIEW

**Comment by kbhawkey** [comment]
See https://kubernetes.io/docs/reference/using-api/api-concepts/#the-resourceversion-parameter

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -11,11 +11,6 @@ $announcement-size-adjustment: 8px;
       max-width: 100%;
     }
 
-    table {
-      margin-bottom: 1rem;
-      display: initial;
-    }
-
```
Comment (tengqm) [inline assets/scss/_custom.scss:18]: Nice. I'm wondering why we have so many customizations ...

**Comment by kbhawkey** [comment]
@irvifa Do you have time to review this pull request?
Thank you!

**Review by irvifa — COMMENTED** [review]
Thanks!
/approve

**Pull request merged**